test/README.md

```diff
@@ -1,69 +1,154 @@
-# Cilium Test Suite
+# Cilium Test Framework
 
-Cilium test suite is based on [Ginkgo test
-framework](https://onsi.github.io/ginkgo/), before start writing test, maybe it
-is a good idea to read [Ginkgo
-Quickstart](https://onsi.github.io/ginkgo/#getting-started-writing-your-first-test)
-guide and run this [test suite
-example](https://github.com/onsi/composition-ginkgo-example) written by the
-[Ginkgo creator](https://github.com/onsi/).
 
-So, before writing test, we should understand the test suite flavors that have
-the system:
+## Introduction 
+The tests in this directory are based off of the [Ginkgo test
+framework](https://onsi.github.io/ginkgo/). Ginkgo provides a rich framework
+for developing tests alongside the benefits of Golang (compilation-time checks,
+types, etc.). To get accustomsed to the basics of Ginkgo, we recommend reading
+the [Ginkgo Getting-Started Guide](https://onsi.github.io/ginkgo/#getting-started-writing-your-first-test),
+as well as running [example tests](https://github.com/onsi/composition-ginkgo-example) 
+to get a feel for the Ginkgo workflow.
 
-- Runtime: This is the flavor where all cilium features are tested.
-- K8S: This is the flavor that we use to test Kubernetes specified features.
-
-Each flavor will power on a new virtual server (vagrant box), ginkgo will
-connect to this server using ssh and run the test specified for that flavor.
-
-To detect what flavor we are going to run, `BeforeSuite` in the test_suite.go
-will identify the scope based on the focus flag string, and it will power on
-the specified server
-
-When the server is power on, the normal execution of the test will start. On
-each spec, a new ssh connection will be open to the virtual server. Mainly this
-connections are wrapped in some helpers, and a Node attribute is exported in
-case that we need to run a specific command on a node.
+The Cilium Ginkgo framework has separate test-suites:
 
+- Runtime: Cilium in a single Vagrant VM managing Docker networking.
```

Comment:
The tests are not limited to docker networking. Let's keep it generic. "Orchestration independent, single node runtime tests".

Reply:
I removed this file in a follow-up commit, I'll squash the commits together to make this more clear.